index.html

```diff
@@ -1,216 +1,236 @@
+        </li>
+        <li>
+          <a href="https://github.com/MohammedElseddik">
+            <img src="./img/github.svg" alt="" /></a>
+        </li>
+        <li>
+          <a href=""><img src="./img/Vector(4).svg" alt="" /></a>
+        </li>
+      </ul>
+      <button type="submit">Get my resume</button>
+    </div>
+
+    <div class="skill-list">
+      <ul class="categories-list">
+        <li>
+          <div>
+            <h2>langguages</h2>
+            <img class="arrow1" src="./img/arrow.svg" alt="" />
+          </div>
+          <ul class="langguages">
+            <li>
+              <img src="./img/javascript-icon.svg" alt="" />
+              <p>JavaScript</p>
+            </li>
+            <li>
+              <img src="./img/Html-icon.svg" alt="" />
+              <p>HTML</p>
+            </li>
+            <li>
+              <img src="./img/css-icon.svg" alt="" />
+              <p>CSS</p>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <div>
+            <h2>Frameworks</h2>
+            <img class="arrow2" src="./img/arrow2.svg" alt="" />
+          </div>
+          <ul class="frameworks"></ul>
+        </li>
+        <li>
+          <div>
+            <h2>Skills</h2>
+            <img class="arrow2" src="./img/arrow2.svg" alt="" />
+          </div>
+          <ul class="skills"></ul>
+        </li>
+      </ul>
+    </div>
+  </section>
+
+  <section class="contact-form-section">
+    <form class="contact-form" action="https://formspree.io/f/xrgjqpdb" method="post">
+      <div class="contact-form__me">
+        <h1>Contact me</h1>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+          needs coding. I’d love to help with it
         </p>
       </div>
-      <div class="contact">
-        <a href=""><p>LET'S CONNECT</p></a>
-        <ul>
-          <li>
-            <a href="https://twitter.com/RebbouhElseddik"
-              ><img src="./img/twitter.svg" alt=""
-            /></a>
-          </li>
-          <li>
-            <a href=""><img src="./img/Vector(1).svg" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="./img/Vector(2).svg" alt="" /></a>
-          </li>
-          <li>
-            <a href="https://github.com/MohammedElseddik">
-              <img src="./img/github.svg" alt=""
-            /></a>
-          </li>
-          <li>
-            <a href=""><img src="./img/Vector(4).svg" alt="" /></a>
-          </li>
-        </ul>
-        <button type="submit">Get my resume</button>
+      <div class="contact-form__info">
+        <input type="text" name="full-name" placeholder="Enter your Full name" maxlength="30" required>
+        <input type="text" name="email" placeholder="Enter your email" required>
+        <textarea name="write-your-message-here" maxlength="500" cols="30" rows="10"
+          placeholder="Write your message here"></textarea>
```

Comment:
- [ ] The form is not supposed to be submitted if one or more of the fields is empty. Right, you are missing the required attribute from the message field. Please add the required attribute in your message field.